ci-operator/config/openshift/rook/openshift-rook-master.yaml

```
@@ -0,0 +1,24 @@
+binary_build_commands: GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
```

```suggestion
    tag: rhel-8-release-golang-1.15-openshift-4.6
```
maybe this has the `shasum` tool required for builds.